TypeScript.py

```diff
@@ -792,6 +795,29 @@ def on_close(self, view):
                     cli.service.close(view.file_name())
         logger.view_debug(view, "exit on_close")
 
+    def handlePulse(self):
+        view = active_view()
+        info = self.getInfo(view)
+        if info:
+            self.showTopLineStatus(info, view)
+        sublime.set_timeout(self.handlePulse, 500)
+
+    def showTopLineStatus(self, info, view):
```

Comment:
by "Top line", do you mean the title of each tab? "Top line" sounds like the first line of the content. It may help if you can put new definitions in the doc string of the method.